blob/fileblob/fileblob.go

```diff
@@ -480,9 +480,11 @@ func (b *bucket) Attributes(ctx context.Context, key string) (*driver.Attributes
 		ContentLanguage:    xa.ContentLanguage,
 		ContentType:        xa.ContentType,
 		Metadata:           xa.Metadata,
+		CreateTime:         info.ModTime(),
 		ModTime:            info.ModTime(),
 		Size:               info.Size(),
 		MD5:                xa.MD5,
+		ETag:               fmt.Sprintf("W/%d", info.ModTime().UnixNano()),
```

Comment:
Check out https://serverfault.com/a/690374 

Basically Nginx set the ETag based on the modification time and size and not as a weak etag. Easy to do here and should probably be better as it is done by a production grade server.

Also ETag has strange mandatory quotes for it to work as far as I remember, e.g.

```
ETag: "XYZ"
Or:
ETag: W/"XYZ"
```

(Probably applies to memblob as well)

Reply:
Thanks! I updated fileblob and memblob to include both mod time and size, and dropped the /W prefix.

I updated the concrete type (`blob`) to note that it strips surrounding quotes (providers are not consistent about whether they include them).